e2e/icons-react/__snapshots__/PublicAPI-test.js.snap

```diff
@@ -1881,6 +1940,8 @@ Array [
   "UpdateNow",
   "Upgrade",
   "Upload",
+  "Upstream",
+  "Url",
```

Comment:
And here?

Reply:
Same for URL, should be all caps I think